### PR TITLE
Track whether agent or member hung up queue call

### PR DESCRIPTION
**Pull request by danielfinke**
#1247 against master

### CODE REVIEW

File: applications/acdc/src/acdc_agent_fsm.erl [hunk 1]
```diff
@@ -936,10 +936,10 @@ answered({'sync_req', JObj}, #state{agent_listener=AgentListener
     {'next_state', 'answered', State};
 answered({'channel_unbridged', CallId}, #state{member_call_id=CallId}=State) ->
     lager:info("caller channel ~s unbridged", [CallId]),
-    {'next_state', 'wrapup', State#state{wrapup_ref=wrapup_timer(State)}};
+    {'next_state', 'answered', State};
 answered({'channel_unbridged', CallId}, #state{agent_call_id=CallId}=State) ->
     lager:info("agent channel unbridged"),
-    {'next_state', 'wrapup', State#state{wrapup_ref=hangup_call(State)}};
+    {'next_state', 'answered', State};
```
Comment (k-anderson) [inline applications/acdc/src/acdc_agent_fsm.erl]: @danielfinke 
This and the diff above do not seem related to tracking the hangup unless I misunderstood something (possible as I don't know ACDc that well).  Are you sure that the transition here should not go to wrapup?  I suspect this might be a fix for something else ;)

Comment (danielfinke) [inline applications/acdc/src/acdc_agent_fsm.erl]: We've been running this code in production for a while and haven't seen a case where this change affects the behaviour of the agent. Do you know of one? As far as the PR, it is important because the bridge always happens to unbridge from the agent side, regardless of who hangs up (which means this feature doesn't work as expected)

Comment (k-anderson) [inline applications/acdc/src/acdc_agent_fsm.erl]: @danielfinke 
Ah, ok that makes sense.  I am not 100% sure of the impact I will need some time to follow the code path.  I suspect it is for offhook agents.  

Once I can verify that it doesn't impact existing functionality the rest of the PR looks good.

**Comment by lazedo** [comment]
@danielfinke Thank You
